x/application/client/cli/tx_delegate_to_gateway.go

```diff
@@ -3,33 +3,42 @@ package cli
 import (
 	"strconv"
 
+	"pocket/x/application/types"
+
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
 	"github.com/spf13/cobra"
-	"pocket/x/application/types"
 )
 
 var _ = strconv.Itoa(0)
 
 func CmdDelegateToGateway() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "delegate-to-gateway",
-		Short: "Broadcast message delegate-to-gateway",
-		Args:  cobra.ExactArgs(0),
+		Use:   "delegate-to-gateway [gateway address]",
+		Short: "Delegate an application to a gateway",
+		Long: `Delegate an application to the gateway whose address is provided. This is a broadcast operation
+that will delegate trust over signing relays from the application to the gateway specified, allowing the gateway
+to sign relay requests on behalf of the application during a session.
```

Comment:
I :heart: to see the long description! :rocket: 

I struggled with the word "over" in the phrase "delegate trust over". With some help from chatGPT I would propose rephrasing like so:

```suggestion
		Long: `Delegate an application to the gateway with the provided address. This is a broadcast operation
that delegates authority to the gateway specified to sign relays requests for the application, allowing the gateway
act on the behalf of the application during a session.
```